### PR TITLE
1010/feature/30792/update insurance question

### PR DESCRIPTION
**Pull request by lzapson**
## Description
As a Veteran, I need to clearly and accurately answer the questions about my Insurance, in order to complete my application for VA Health Care.

Currently, the research shows that this question causes confusion, and people are having to guess what information to input. This can result in inaccurate information being submitted.

- Added text and accordion info on insurance provider page
- changed error text  

## Original issue(s)
[department-of-veterans-affairs/va.gov-team#0000](https://github.com/department-of-veterans-affairs/va.gov-team/issues/34577)

## Testing done
updated unit tests
updated e2e tests
axe accessibility test

## Screenshots

other-coverage
other-coverage-with-errors

### CODE REVIEW

**Review by mdewey — APPROVED** [review]
LGTM

**Review by timwright12 — APPROVED** [review]

**Review by timwright12 — APPROVED** [review]

**Review by timwright12 — APPROVED** [review]

File: src/applications/hca/config/chapters/insuranceInformation/general.js [hunk 1]
```diff
@@ -1,53 +1,107 @@
...
         viewField: InsuranceProviderView,
       },
       'ui:errorMessages': {
         minItems: 'You need to at least one provider.',
       },
       items: {
         insuranceName: {
-          'ui:title': 'Name of provider',
+          'ui:title': 'Name of insurance provider',
+          'ui:autofocus': true,
```
Comment (Mottie) [inline src/applications/hca/config/chapters/insuranceInformation/general.js:37]: I'm not sure if `autofocus` is okay to use here. We generally focus on the `h1` or breadcrumbs when going to a new page in a form, but I would definitely ask an accessibility expert about it.

File: src/applications/hca/config/chapters/insuranceInformation/general.js [hunk 2]
```diff
@@ -1,53 +1,107 @@
...
+              <p className="vads-u-color--gray vads-u-margin-top--0 vads-u-margin-bottom--0">
+                Either this or the group code is required
+              </p>
+            </div>
+          ),
+        },
+        'view:or': {
+          'ui:description': (
+            <div className="schemaform-block-title schemaform-block-subtitle vads-u-margin-bottom--neg2p5">
+              <strong>or</strong>
```
Comment (Mottie) [inline src/applications/hca/config/chapters/insuranceInformation/general.js:70]: I don't know the context of this page, but in bulleted lists (I know this isn't a bulleted list), the content guidelines say to add a `, <strong>or</strong>` to the end of the line - see https://design.va.gov/content-style-guide/bulleted-lists.

File: src/applications/hca/helpers.jsx [hunk 3]
```diff
@@ -498,6 +498,63 @@ export const isEssentialAcaCoverageDescription = (
     </div>
   </div>
 );
+export const healthInsuranceCoverageQuestionDescription = (
+  <div className="vads-u-margin-bottom--3">
+    <div className="hca-tooltip-wrapper">
+      <AdditionalInfo triggerText="Why we ask this information">
```
Comment (Mottie) [inline src/applications/hca/helpers.jsx:504]: We should be using `<va-additional-info trigger="...">` instead

File: src/applications/hca/config/chapters/insuranceInformation/general.js (hunk 2 above)
Comment (lzapson) [reply src/applications/hca/config/chapters/insuranceInformation/general.js]: Removed the <strong> tag as not needed with classes; this is not a bulleted list.

File: src/applications/hca/helpers.jsx (hunk 3 above)
Comment (lzapson) [reply src/applications/hca/helpers.jsx]: changed as indicated, ty

File: src/applications/hca/config/chapters/insuranceInformation/general.js (hunk 1 above)
Comment (lzapson) [reply src/applications/hca/config/chapters/insuranceInformation/general.js]: removed, meant to remove before commit, thanks for all your feedback, I'm new at Ad Hoc, this is my first pull request.

**Pull request merged**